src/System.Runtime.Analyzers/Core/TestForNaNCorrectly.cs

```diff
@@ -27,7 +27,7 @@ public sealed class TestForNaNCorrectlyAnalyzer : DiagnosticAnalyzer
                                                                              s_localizableMessage,
                                                                              DiagnosticCategory.Usage,
                                                                              DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: true,
+                                                                             isEnabledByDefault: false,
```

Comment:
Ah, thanks for catching this. This analyzer was crashing before this package upgrade and I had turned it off locally. Did not intend to include this - I'll revert.